File: net/Link.Message.Client/content/complex/linear/TextLinearComplexMessageContentItem.cs
using Link.Message.Client.utils;

namespace Link.Message.Client.content.complex.linear
{
	/// <summary>
	/// Hint类型的线性消息内容项
	/// </summary>
	public class TextLinearComplexMessageContentItem : TitleLinearComplexMessageContentItem
	{
		public TextLinearComplexMessageContentItem(string hint)
		{
			Guard.GuardReqiredString(hint, "hint must be set value.");

			Content = new StringContent(hint);
			DisplayType = LinearComplexMessageContentType.Text;
		}
	}
}

File: java/link-message-client/src/main/java/link/message/client/content/complex/linear/TextLinearComplexMessageContentItem.java
package link.message.client.content.complex.linear;

import link.message.client.content.StringContent;
import link.message.client.utils.Guard;

/**
 * Hint类型的线性消息内容项
 * 
 * @author zhong_t
 */

public class TextLinearComplexMessageContentItem extends TitleLinearComplexMessageContentItem {
	public TextLinearComplexMessageContentItem(String hint) {
		Guard.guardReqiredString(hint, "hint must be set value.");
		
		this.content = new StringContent(hint);
		this.displayType = LinearComplexMessageContentType.TEXT;
	}
}